Bioimage-io-scripts/UNet3DKinetochores.model.yaml

```diff
@@ -48,11 +48,11 @@ outputs:
     axes: bczyx
     data_type: float32
     data_range: [-inf, inf]
-    halo: [0, 0, 32, 48, 48]
+    halo: [0, 32, 48, 48]
```

Comment:
is really so much of the returned output affected by edge artefacts?

Reply:
This is based on the settings in Pytorch-3DUNet, which worked well for Kinetochores use case (training as well as inference). But do you suggest to go for smaller values? (I can try that)

Reply:
could this be the halo within the network? You have your slicer, etc. to go over the whole volume. Just make sure that this is actually specifying the final output and not an intermediate step within your algorithm. If that's the case leave it as is and let's get this working before we start tweaking things.

Reply:
Yes, it is part of the predictor [config](https://github.com/wolny/pytorch-3dunet/blob/ashwin/resources/test_dice_egfp.yaml#L48) and the [routine](https://github.com/wolny/pytorch-3dunet/blob/ashwin/pytorch3dunet/unet3d/predictor.py#L89).

Reply:
my understanding was that as this slicer is only used within your model it has no direct influence over the overall in- and output of the whole bioimage.io model. Let's take a closer look at this when we have a more or less running example.